grails-app/services/smartR/plugin/SmartRService.groovy

```diff
@@ -116,4 +117,11 @@ class SmartRService {
 
         return scriptExecutorService.run(parameterMap)
     }
+
+
+    def getScriptsFolder(){
+        def scriptsFolderName = "HeimScripts"
+        def scriptFolder = "/${scriptsFolderName}/".toString() //In order to pass Java String instead of GString
+        return getWebAppFolder() +scriptFolder
+    }
```

Comment:
GString should be converted to the String when you will declare the return type of the method explicitly `String getScriptsFolder()`
Then you could return simply "${webAppFolder}/${scriptsFolderName}/"